app/views/user_journey/publish_failed.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= page_title t('user_journey.confirmation.publish_failed_title') %>
+
+<%= render 'shared/certificate_details', certificate: @certificate %>
+
+<h1 class="govuk-heading-l"><%= t('user_journey.confirmation.failed_to_publish_heading') %></h1>
+<p><%= t('user_journey.confirmation.please_contact') %> <a href="mailto:<%=t('user_journey.confirmation.support_email') %> class="govuk-link"><%= t('user_journey.confirmation.support_email')%></a>.</p>
+<p><%= t('user_journey.confirmation.team_help_publish') %> <%= @certificate.component.display %> <%= @certificate.usage %> certificate.</p>
```

Comment:
this should use proper string interpolation, e.g.
```suggestion
<p><%= t('user_journey.confirmation.team_help_publish'), component: @certificate.component.display, usage: @certificate.usage %> </p>
```

Reply:
Thanks, have amended.